armbian/mender-convert.sh

```diff
@@ -71,13 +71,17 @@ case ${ACTION} in
 		rm "output/${TEMP_NAME}.ext4"
 
 		mkdir -p "../../bin/img-mender/${VERSION}"
-		mv "output/${TEMP_NAME}.sdimg" "output/${TARGET_NAME}.img"
-		mv "output/${TEMP_NAME}.mender" "output/${TARGET_NAME}.base-unsigned"
-		mv output/"${TARGET_NAME}"* "../../bin/img-mender/${VERSION}/"
+
+		cd output
+		mv "${TEMP_NAME}.sdimg" "${TARGET_NAME}.img"
+		tar -czf "${TARGET_NAME}.tar.gz" "${TARGET_NAME}.img"
```

Comment:
nit: `.tar.gz` or tarball usually indicates a compressed folder. This is only a single compressed file.

Reply:
Good point, but will it leave for now.